src/ResourceManager/RecoveryServices/Commands.RecoveryServices.SiteRecovery/help/Edit-AzureRmRecoveryServicesAsrRecoveryPlan.md

```diff
@@ -56,7 +56,7 @@ Appends a group to existing Azure Site Recovery recovery plan and returns the in
 List of ASR replication protected items to be added to the recovery plan group in the recovery plan object.
 
 ```yaml
-Type: Microsoft.Azure.Commands.RecoveryServices.SiteRecovery.ASRReplicationProtectedItem[]
+Type: ASRReplicationProtectedItem[]
```

Comment:
You'll have to regenerate all your help files. You forgot to add the `-UseFullTypeName` switch. See the documentation here: https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md#updating-a-single-markdown-file

Reply:
This is something new  from what we are doing in the way to generate help file.
will do the changes follow this from now onwards